src/ButtonLink/ButtonLink.js

```diff
@@ -0,0 +1,212 @@
+// @flow
+import * as React from "react";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import styled from "styled-components";
+
+import { iconSizes } from "../Icon";
+
+export const TYPES = {
+  PRIMARY: "primary",
+  SECONDARY: "secondary",
+};
+
+export const SIZES = {
+  SMALL: "small",
+  NORMAL: "normal",
+  LARGE: "large",
+};
+
+type Props = {
+  children?: React.Node,
+  component: string | React.Node,
+  onClick?: (e: SyntheticEvent<HTMLButtonElement>) => void,
+  disabled: boolean,
+  block: boolean,
+  external: boolean,
+  type: $Values<typeof TYPES>,
+  size: $Values<typeof SIZES>,
+  href?: string,
+  width: number,
+  icon?: React.Node,
+  theme: typeof defaultTokens,
+  transparent?: boolean,
```

Comment:
We can define `transparent` as required and set default to `false`

Reply:
Yes, but  i don't see any benefits, the same we can do with other props, but why ?

Reply:
Currently, you can see `transparent={false}` in the preview source code.
Also, if someone will copy this code, there is no need to pass `anyProp={false}` to a component.

Reply:
It'll probably results in warning/error I think.